quint/src/parsing/ToIrListener.ts

```diff
@@ -85,8 +86,6 @@ export class ToIrListener implements QuintListener {
   protected exprStack: QuintEx[] = []
   // the stack of parameter lists
   protected paramStack: QuintLambdaParameter[] = []
-  // stack of names used as parameters and assumptions
-  protected identOrHoleStack: string[] = []
```

Comment:
We don't need this stack, because these are just string values we can grab directly.